card.js

```diff
@@ -93,8 +101,13 @@ class RGBLightCard extends HTMLElement {
     }
 
     applyColor(color) {
-        const serviceData = { entity_id: this.config.entity, ...color, icon_color: undefined, type: undefined };
-        this._hass.callService(color.type || 'light', 'turn_on', serviceData);
+        if (color.type === 'call-service') {
+            const [domain, service] = color.service.split('.');
+            this._hass.callService(domain, service, color.service_data);
+        } else {
+            const serviceData =  { entity_id: this.config.entity, ...color, icon_color: undefined, type: undefined };
```

Comment:
Extra space needs removing between the = and {
```suggestion
            const serviceData = { entity_id: this.config.entity, ...color, icon_color: undefined, type: undefined };
```

Reply:
Actually, you just need to run `npm run lint` to fix style, and commit again